c/extras/aggregator.h

```diff
@@ -41,6 +41,7 @@ class Aggregator {
     int32_t nd_bins;
     int32_t max_dimensions;
     unsigned int seed;
+    int : 32;
```

Comment:
Is it to suppress the warning? I've tried to re-arrange the fields, but it didn't help.